pkgs/servers/keycloak/default.nix

```diff
@@ -7,7 +7,7 @@
 , callPackage
 , confFile ? null
 , plugins ? [ ]
-, extraFeatures ? [ ]
+, extraFeatures ? [ "hostname:v1" ]
```

Comment:
Looks like this needs to be 
```suggestion
, extraFeatures ? [ "hostname-v1" ]
```

Reply:
This fails even earlier, at the build time.

Reply:
It's failing [here](https://github.com/keycloak/keycloak/blob/25.0.1/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/hostname/DefaultHostnameProvider.java#L310), might need to set `hostname-port`?

Reply:
Doesn't work either.